package.json

```diff
@@ -87,6 +87,7 @@
     "typescript": "^4.9.4"
   },
   "peerDependencies": {
+    "easymde": ">= 2 < 3",
```

Comment:
Isn't it better to do
```suggestion
    "easymde": "^2",
```
To accomplish the same thing?

Reply:
Thats the semver range used by the react lib, I feel we should mirror it